dev/run-tests.py

```diff
@@ -342,7 +342,7 @@ def build_spark_unidoc_sbt(extra_profiles):
     set_title_and_block("Building Unidoc API Documentation", "BLOCK_DOCUMENTATION")
     # Enable all of the profiles for the build:
     build_profiles = extra_profiles + modules.root.build_profile_flags
-    sbt_goals = ["unidoc"]
+    sbt_goals = ["clean", "unidoc"]
```

Comment:
This is not relevant to this PR. I am trying if the jenkins tests are passed with it.

Reply:
Nice. It passed now. Could you make a new PR for this line first?

Reply:
Sure